client/src/components/TextInputComponent/TextInput.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+
+import TextInput from "./TextInput";
+
+storiesOf("TextInput", module)
+  .add("Primary", () => (
+    <TextInput labelName={""} type={"text"} placeholder={"Sign up here"} />
```

Comment:
You can show that the `label` prop is optional by leaving it off here

Reply:
Absolutely! That was a question I had, logically I should have already known that answer now.